nasa-gcn/gcn.nasa.gov
# Close Circular and Schema Version Dropdowns When Clicking Away

# Description
This adds the functionality from #1989 to the other two dropdown menus located in the repo at the moment: the version dropdown menus for the Schema browser and any circulars with multiple versions. 

The method for determining if the dropdown content is shown or hidden for these has also been updated to match the other two dropdowns.

# Related Issue(s)

Resolves #2345 

# Testing
Schema Browser Dropdown:
1. Navigate to Docs page
2. Select 'Schema Browser' in menu
3. Click on Version dropdown in upper right-hand corner to expand the dropdown content
4. Clicking on the button again should collapse the dropdown content, and clicking elsewhere on the page while the dropdown is expanded should also collapse it

Circulars Dropdown:
1. Navigate to a Circular with multiple versions (example: 34776)
2. Click on Version dropdown in upper right-hand corner to expand the dropdown content
3. Clicking on the button again should collapse the dropdown content, and clicking elsewhere on the page while the dropdown is expanded should also collapse it
